app/src/app/user/profile/profile.tpl.html

```diff
@@ -93,15 +93,76 @@
             <h3>Preferences</h3>
             <md-whiteframe class="md-whiteframe-z1" layout="column" layout-padding>
                 <md-select-label>TIMEZONE</md-select-label>
-                <md-select ng-model="ProfileController.user.timezoneIdentifier" aria-label="Timezone">
+                <md-select ng-model="ProfileController.fullUserInfo.timezoneIdentifier" aria-label="Timezone">
                     <md-option ng-value="timezone.timezoneIdentifier" ng-repeat="timezone in ProfileController.timezones">{{ timezone.timezoneIdentifier }}</md-option>
                 </md-select>
             </md-whiteframe>
         </md-content>
 
+        <md-content layout-padding>
+            <h3>Access</h3>
+            <md-whiteframe class="md-whiteframe-z1" layout="column" layout-padding>
+                <label>LOGIN METHODS</label>
+                <md-list>
+                    <md-list-item class="md-2-line" ng-repeat="providerType in ProfileController.providerTypes">
+                        <md-icon>accessibility</md-icon>
+                        <div class="md-list-item-text">
+                            <h3>{{ providerType }}</h3>
+                            <h4>{{ ProfileController.fullUserInfo.hasSocialLogin(providerType) ? 'Connected as: ' + ProfileController.fullUserInfo.fullName() : 'Not connected' }}</h4>
+                        </div>
+                        <md-button ng-if="ProfileController.fullUserInfo.hasSocialLogin(providerType)" ng-click="ProfileController.unlinkSocialLogin(providerType)" class="md-raised" type="button">
```

Comment:
@xiphiaz, I have still opted to use ternary operators at this point as this was the cleanest solution I could implement. Using conditionals such as ng-if, ng-switch added in divs which were unnecessary and would cause the layout to go funny (is there anyway to implement these conditionals without adding in div blocks?). I'm open to suggestions although I think it may be better just to leave it at this point and come back to it when styles are applied because it could change again.